### PR TITLE
update setup-api readme

### PR DESCRIPTION
**Pull request by RomKadria**

### CODE REVIEW

**Review by AvnerMaster — COMMENTED** [review]
I approved, but how did you found out that this command: `npx @mondaydotcomorg/setup-api` works only on linux shell?

**Review by AvnerMaster — CHANGES_REQUESTED** [review]
I approved, but how did you found out that this command: npx @mondaydotcomorg/setup-api works only on linux shell?

File: packages/setup-api/README.md [hunk 1]
```diff
@@ -1,5 +1,15 @@
 # Monday API Setup Library
 
+**Important: This command works only in bash-like environments (e.g., macOS Terminal, iTerm, Linux shell, Git Bash on Windows). It is not compatible with Windows Command Prompt (cmd).**
+
+## Installation
+
+Run this command on the **root** directory of your project
+
+```bash
+npx @mondaydotcomorg/setup-api
```
Comment (AvnerMaster) [inline packages/setup-api/README.md:10]: I approved, but how did you found out that this command: npx @mondaydotcomorg/setup-api works only on linux shell?

**Review by AvnerMaster — APPROVED** [review]

File: packages/setup-api/README.md (hunk 1 above)
Comment (RomKadria) [reply packages/setup-api/README.md:10]: I always knew it. if you look at the code for install youl also see the modification done for making the shell work with windows pc

**Pull request merged**